src/public/js/api.js

```diff
@@ -413,6 +413,25 @@ const renderReceipt = (receipt) => {
       amountFormatter,
       ['']
     );
+    // add card details
+    receipt.payments.forEach((payment) => {
+      if (payment.type === 'card') {
+        buildItems(
+          mainPaymentsList,
+          [
+            { label: 'last four', value: payment.lastFour },
+            { label: 'auth code', value: payment.authCode }
+          ],
+          [''],
```

Comment:
I'm reluctant to admit; but i don't really understand whats going on here; why does buildItems take a whole bunch of lists and the same propFn value a bunch of times.